src/components/DeleteKebab/DeleteKebab.tsx

```diff
@@ -48,6 +52,17 @@ const DeleteKebab = ({
           : 'Remove selected repositories'}
       </DropdownItem>
     </ConditionalTooltip>,
+
+    <ConditionalTooltip
+      key='snapshot'
+      content='Manually trigger a snapshot in the repository'
+      show={true}
+      setDisabled={false}
+    >
+      <DropdownItem onClick={triggerSnapshot} isDisabled={!rbac?.write || isDisabled}>
+        Trigger Snapshot
+      </DropdownItem>
+    </ConditionalTooltip>,
```

Comment:
Oh actually, this is completely the wrong file. 
Revert all changes, rebase from master, and attempt to make these changes in the kebab seen at the end of the row on the seen here: 
![Screenshot 2023-12-04 at 1 34 54 PM](https://github.com/content-services/content-sources-frontend/assets/38083295/d936726c-5530-4989-a2a6-c7002e728a3c)


Your current changes, are here: 
![Screenshot 2023-12-04 at 1 35 25 PM](https://github.com/content-services/content-sources-frontend/assets/38083295/11eb0c4a-0a31-4138-9107-e7507e102e82)

This results in your changes being in the incorrect spot on two pages. 
![Screenshot 2023-12-04 at 1 35 59 PM](https://github.com/content-services/content-sources-frontend/assets/38083295/8437c6ae-68b0-4129-bba6-0c5c328f7420)